src/pipeline_manager.py

```diff
@@ -122,35 +128,32 @@ def generate_prediction(img_ids, pipeline, chunk_size):
 
 
 def _generate_prediction(img_ids, pipeline):
-    data = {'input': {'img_ids': img_ids
+    data = {'input': {'img_ids': img_ids,
+                      'image_size': (256, 256)
                       },
             }
 
     pipeline.clean_cache()
     output = pipeline.transform(data)
     pipeline.clean_cache()
-    y_pred = output['y_pred']
-
-    prediction = create_annotations(img_ids, y_pred)
-    return prediction
+    return output['y_pred']
 
 
 def _generate_prediction_in_chunks(img_ids, pipeline, chunk_size):
     prediction = []
     for img_ids_chunk in generate_data_frame_chunks(img_ids, chunk_size):
-        data = {'input': {'img_ids': img_ids_chunk
+        data = {'input': {'img_ids': img_ids_chunk,
+                          'image_size': (256, 256)
```

Comment:
@apyskir I think it is better to have it in pipeline_config.py as it is just one tuple

Reply:
@jakubczakon I am not sure if it is a good idea. This one should be parametrised together with target image size for retina net, because this is actually what it is. But temporarily I can put it into pipeline_config.py